[Term Entry] Python Strings: .rjust() (#3341)
Codecademy/docs: content/python/concepts/strings/terms/rjust/rjust.md

---
Title: '.rjust()'

> Description: 'Adds padding to the left of the parameter string.'

Reviewer: ```suggestion
Description: 'Adds padding to the left of the given string.'
```

Subjects:
  - 'Data Science'
  - 'Computer Science'
Tags:
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> The method **`.rjust()`** adds padding of a specified length to the left of a parameter string. The user can specify a character to use as the padding. The number of padding characters added is equal to the length specified in the method call minus the length of the parameter string.

Reviewer: ```suggestion
The method **`.rjust()`** adds padding of a specified length to the left of a given string. The user can specify a character to use as the padding. The number of padding characters added is equal to the length specified in the method call minus the length of the string provided.
```

## Syntax

```pseudo
parameter_string.rjust(length, padding_character)
```

### Required Parameters

>  - `parameter_string`: This is the string on which padding will be added.

Reviewer: ```suggestion
 - `parameter_string`: The string to be modified with padding.
```

 - `length`: An integer used to determine how many characters of padding will be added. The number of padding characters will be equal to `length` minus the length of `string_parameter`.

### Optional Parameters

 - `padding_character`: The character used as the padding. Must be a single character. Default value is a space (`" "`).

## Example

> 

Reviewer: There should be a brief introductory sentence after the example header

Author: Added.
